### DISCUSSION TITLE
Comparison Between Scale and ZLUDA: CUDA on AMD GPUs

### DISCUSSION THREAD

**Discussion by ykdu** [vosen/ZLUDA#290] [General]
Scale is another solution similar to ZLUDA for running CUDA applications on AMD GPUs. Both Scale and ZLUDA's key contributions lie in their CUDA-compatible dynamic libraries and PTX compilers.

I came across the Scale documentation here: [Scale Documentation](https://docs.scale-lang.com/).

Could someone explain the relationship and differences between Scale and ZLUDA?

**Comment by vosen** [comment]
SCALE documents the differences: https://docs.scale-lang.com/manual/comparison/#zluda
